.erda/migrations/orchestrator/20220621-runtime-service-hpa-rule-base.sql

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2021 Terminus, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE TABLE `ps_v2_runtime_hpa`
```

Comment:
please rename the .sql filename and make sure it dose not has suffix "-base", and do lint on the .sql file.

Reply:
Change the file name and run  make miglint  verified OK.